app/_locales/en/messages.json

```diff
@@ -110,6 +110,43 @@
     "message": "I have read and agree to the $1",
     "description": "$1 is the `terms` message"
   },
+  "accept": {
+    "message": "Accept"
+  },
+  "agreeTermsOfUse":{
+    "message": "I agree to Metamask's $1",
+    "description": "$1 is the `terms` link"
+  },
+  "termsOfUseTitle":{
+    "message": "Our Terms of Use have updated"
+  },
+  "termsOfUse1:":{
```

Comment:
Kevin spoke with Legal and they'd like the Terms to be embedded as English, so we don't need to use this file to translate the `termsOfUse{#}`.  You can embed them directly in the component.